### PR TITLE
Fluent string fixes for Relay migration

### PR DESCRIPTION
**Pull request by stephaniehobson**
https://github.com/mozilla-l10n/www-l10n/pull/343#pullrequestreview-1548350918

### CODE REVIEW

File: l10n/en/products/relay/bundle.ftl [hunk 1]
```diff
@@ -8,7 +8,7 @@
 #   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
 bundle-banner-header-2 = Limited-time only: { -brand-name-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month
 # Variables:
-#   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
+#   $savings (string) - the percentage saved for a given plan, not including the % Examples: 50, 70
 bundle-banner-body-3 = { -brand-name-mozilla-vpn } protects you from tracking & surveillance while you’re gaming, online banking, or getting work done. Add it to { -brand-name-firefox-relay } for { $savings } off.
```
Comment (bcolsson) [inline l10n/en/products/relay/bundle.ftl:12]:
```suggestion
bundle-banner-body-3-v2 = { -brand-name-mozilla-vpn } protects you from tracking & surveillance while you’re gaming, online banking, or getting work done. Add it to { -brand-name-firefox-relay } for { $savings }% off.
```
We'll want to have the % in our string now.

File: l10n/en/products/relay/matrix.ftl [hunk 2]
```diff
@@ -9,6 +9,8 @@ plan-matrix-title = Pricing
 ## PRICING MATRIX SECTION
 
 plan-matrix-offer-title = Choose a level of protection that’s right for you
+# Variables:
+#   $savings (string) - the percentage saved for a given plan, not including the % Examples: 50, 70
 plan-matrix-offer-body = { -brand-name-firefox-relay } protects your identity and your inbox with unique email masks. Add on even more security with { -brand-name-relay } phone masking, and save { $savings } by adding { -brand-name-vpn } protection.
```
Comment (bcolsson) [inline l10n/en/products/relay/matrix.ftl:14]:
```suggestion
plan-matrix-offer-body-v2 = { -brand-name-firefox-relay } protects your identity and your inbox with unique email masks. Add on even more security with { -brand-name-relay } phone masking, and save { $savings }% by adding { -brand-name-vpn } protection.
```

File: l10n/en/products/relay/matrix.ftl [hunk 3]
```diff
@@ -60,6 +62,8 @@ plan-matrix-price-period-yearly = Yearly
 plan-matrix-price-period-monthly = Monthly
 plan-matrix-price-period-yearly-footnote-1 = Billed yearly
 plan-matrix-price-period-monthly-footnote-1 = Billed monthly
+# Variables:
+#   $savings (string) - the percentage saved for a given plan, not including the % Examples: 50, 70
 plan-matrix-price-vpn-discount-promo = <span>Save { $savings }</span> on regular { -brand-name-vpn } price
```
Comment (bcolsson) [inline l10n/en/products/relay/matrix.ftl:67]:
```suggestion
plan-matrix-price-vpn-discount-promo-v2 = <span>Save { $savings }%</span> on regular { -brand-name-vpn } price
```

**Review by bcolsson — CHANGES_REQUESTED** [review]

**Review by craigcook — APPROVED** [review]
💾

**Pull request merged**